torchvision/prototype/transforms/functional/_geometry.py

```diff
@@ -608,6 +608,9 @@ def pad(
     if isinstance(inpt, features._Feature):
         return inpt.pad(padding, fill=fill, padding_mode=padding_mode)
     elif isinstance(inpt, PIL.Image.Image):
+        if fill is None:
+            fill = 0
+
```

Comment:
Is this needed? PIL's pad signature seems to suggest that is handles None:
https://github.com/pytorch/vision/blob/51f27d766846e8347d4b56b95434f02950fe9e2b/torchvision/transforms/functional_pil.py#L146-L149

Reply:
Actually, this type hint is wrong due to the internal type check: 
https://github.com/pytorch/vision/blob/2e70ee1af3bb3c5ba0f9a8bcf16cd1611831690f/torchvision/transforms/functional_pil.py#L158-L159
so, fill can't be `None`

Reply:
Do you think we should be handling it inside the pad method? Aka assign 0 to it.

Reply:
Yes, we can. Let me do that